test/sqllogictest/transform/predicate_pushdown.slt

```diff
@@ -72,10 +72,7 @@ EXPLAIN PLAN FOR
 WHERE b = 3 AND c = 3
 ----
 %0 =
-| Get materialize.public.y (u3)
-| Filter (3 = (#0 + 1))
-| Map (#0 + 1), (#1 + 1)
-| Filter (#2 = 3)
+| Constant
```

Comment:
Is the comment about this query on line 64 still relevant?

Reply:
I guess not much. We could add a query that doesn't result in an empty relation instead so that what the comment says is still tested explicitly.